docs/book/src/reference/version-skew-policy.md

```diff
@@ -1,60 +1,68 @@
+|Kubernetes v1.13|✓||
+|Kubernetes v1.14|✓||
+|Kubernetes v1.15|✓||
+|Kubernetes v1.16|✓|✓|
+|Kubernetes v1.17||✓|
+|Kubernetes v1.18||✓|
 
-The Core Provider also talk to the Kubernetes API of every Workload Cluster. The Workload Cluster's Kubernetes API version must also be compatible.
+The Core Provider also talks to the Kubernetes API of every Workload Cluster. The Workload Cluster's Kubernetes API version must also be compatible.
 
 In some cases, the Management Cluster is separate from the Workload Clusters. The Kubernetes API version of the Management and Workload Clusters are allowed to be different. For instance, the Management Cluster can be v1.18.2, and the Workload Clusters can be at v1.16.9 and v1.17.5.
 
 ### Kubeadm Bootstrap Provider - `kubeadm-bootstrap-controller`
 
-||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
-|-|-|-|-|
-|Kubernetes v1.13||||
-|Kubernetes v1.14||✓||
-|Kubernetes v1.15||✓||
-|Kubernetes v1.16||✓|✓|
-|Kubernetes v1.17|||✓|
-|Kubernetes v1.18|||✓|
+||Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
+|-|-|-|
+|Kubernetes v1.13|||
+|Kubernetes v1.14|✓||
+|Kubernetes v1.15|✓||
+|Kubernetes v1.16|✓|✓|
+|Kubernetes v1.17||✓|
+|Kubernetes v1.18||✓|
 
 The Kubeadm Bootstrap Provider generates configuration using the kubeadm APIs.
 
 ### Kubeadm Control Plane Provider - `kubeadm-control-plane-controller`
 
-||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
-|-|-|-|-|
-|Kubernetes v1.13||||
-|Kubernetes v1.14||||
-|Kubernetes v1.15||||
-|Kubernetes v1.16 + etcd v3|||✓|
-|Kubernetes v1.17 + etcd v3|||✓|
-|Kubernetes v1.18 + etcd v3|||✓|
+||Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
+|-|-|-|
+|Kubernetes v1.13|||
+|Kubernetes v1.14|||
+|Kubernetes v1.15|||
+|Kubernetes v1.16 + etcd v3||✓|
+|Kubernetes v1.17 + etcd v3||✓|
+|Kubernetes v1.18 + etcd v3||✓|
 
 The Kubeadm Control Plane Provider talks to the Kubernetes and etcd APIs of every Workload Cluster whose control plane it owns.
 
 ### Infrastructure Providers
 
-Infrastructure Providers are _independently supported_. An Infrastructure Provider that is compatible with Cluster API version N must be compatible with at least one Kubernetes API version supported by Cluster API version N. An Infrastructure Provider may support a smaller set of versions than Cluster API version N.
+Infrastructure Providers are _independently maintained_. If an Infrastructure Provider version M is compatible with Cluster API version N, then M must be compatible with at least one Kubernetes API version supported by Cluster API version N. The Infrastructure Provider version M may be compatible with a smaller set of Kubernetes versions than Cluster API version N.
 
 For version compatibility, refer to each Infrastructure Provider's documentation. See this list of [public Infrastructure Providers](providers.md).
 
 ### Visual Representation
 
 #### Management And Workload Cluster Are the Same
```

Comment:
```suggestion
#### Management And Workload Cluster Are the Same (Self-hosted)
```

Reply:
I'll change this, thanks

Reply:
I saw your comments just after I pushed an update. I think it addresses the feedback you had for Infrastructure Providers.